dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -260,4 +260,46 @@ interface CompanyApi {
         produces = ["application/json"],
     )
     fun getTeaserCompanies(): List<String>
+
+    /**
+     * A method used to retrieve the aggregated meta information for all frameworks
```

Comment:
Should we change the description to something like:
A method used to retrieve the number of available datasets for the specified company aggregated by available dataland framework

Reply:
Yes this was outdated. I changed it.

Reply:
I like the new description you've pushed